### PR TITLE
Update UserGuide, DeveloperGuide and AboutUs

### PR DESCRIPTION
**Pull request by seanpzk**
### Updated the following:

**UserGuide.md :** 
- Table of Contents
- Features > Edited Command Format
- Features > Adding a list of student
- Features > Removing a student
- Features > Exiting the program
- Features > Saving the data
- Command Summary > Added "adding a list of student" for **add** action

**DeveloperGuide.md:**
- Use case > Remove a student
- Use case > Modify an existing student record
- Use case > Use auto-save feature
- Glossary > Database 

**AboutUs.md :** Added my picture and github link

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -11,12 +11,11 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Sean Phua
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/sean.png" width="200px">
```
Comment (songgthu) [inline docs/AboutUs.md]: I think they required the name of the file to be your github username too.

File: docs/AboutUs.md [hunk 2]
```diff
@@ -11,12 +11,11 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
+[[github](https://github.com/seanpzk)]
 [[portfolio](team/johndoe.md)]
 
 * Role: Project Advisor
```
Comment (DonovanJJ) [inline docs/AboutUs.md]: Perhaps Team Lead might be a better representation of your role

Team lead: Responsible for overall project coordination.

File: docs/AboutUs.md (hunk 1 above)
Comment (seanpzk) [reply docs/AboutUs.md]: Ohh, thank you I missed out on that

File: docs/AboutUs.md (hunk 2 above)
Comment (seanpzk) [reply docs/AboutUs.md]: Oops okay thanks

**Review by DonovanJJ — APPROVED** [review]
Looks good to me!

**Review by sudarshan2401 — APPROVED** [review]
LGTM!

**Pull request merged**